frontend/www/js/omegaup/components/problem/SettingsSummary.vue

```diff
@@ -0,0 +1,132 @@
+<template>
+  <div class="panel">
+    <h1 class="title">
+      {{ title }}
+      <template v-if="!inArena">
+        <img
+          src="/media/quality-badge-sm.png"
+          v-bind:title="T.wordsHighQualityProblem"
+          v-if="problem.quality_seal || problem.visibility === 3"
+        />
+        <span
+          class="glyphicon glyphicon-warning-sign"
+          v-bind:title="T.wordsWarningProblem"
+          v-if="problem.visibility === 1 || problem.visibility === -1"
+        ></span>
+        <span
+          class="glyphicon glyphicon-eye-close"
+          v-bind:title="T.wordsPrivate"
+          v-if="problem.visibility === 0 || problem.visibility === -1"
+        ></span>
+        <span
+          class="glyphicon glyphicon-ban-circle"
+          v-bind:title="T.wordsBannedProblem"
+          v-if="problem.visibility <= -2"
+        ></span>
+        <template v-if="isAdmin">
+          (<a href="/problem/{$problem_alias}/edit/">{{ T.wordsEdit }}</a
+          >)
+        </template>
+      </template>
+    </h1>
+    <table class="data">
+      <tr>
+        <td>{{ T.wordsPoints }}</td>
+        <td>{{ problem.points }}</td>
+        <td>{{ T.arenaCommonMemoryLimit }}</td>
+        <td data-memory-limit>{{ memoryLimit }}</td>
+      </tr>
+      <tr>
+        <td>{{ T.arenaCommonTimeLimit }}</td>
+        <td>{{ timeLimit }}</td>
+        <td>{{ T.arenaCommonOverallWallTimeLimit }}</td>
+        <td>{{ overallWallTimeLimit }}</td>
+      </tr>
+      <tr>
+        <template v-if="inArena">
+          <td>{{ T.wordsInOut }}</td>
+          <td>{{ T.wordsConsole }}</td>
+        </template>
+        <td>{{ T.problemEditFormInputLimit }}</td>
+        <td>{{ inputLimit }}</td>
+      </tr>
+    </table>
+  </div>
+</template>
+
+<style lang="scss">
+#problem {
+  .title {
+    text-align: center;
+    font-size: 1.5em;
+    margin: 1em;
+  }
+  .data {
```

Comment:
```suggestion
  table {
```

si se le agrega `scoped` a L57, ya no hay necesidad de crear una clase!

Reply:
Listo.